src/formio/components/textarea.ts

```diff
@@ -0,0 +1,25 @@
+import {InputComponentSchema, MultipleCapable} from '..';
+
+type Validator = 'required' | 'maxLength' | 'pattern';
+type TranslatableKeys = 'label' | 'description' | 'tooltip' | 'defaultValue' | 'placeholder';
+
+export type TextareaInputSchema = InputComponentSchema<string, Validator, TranslatableKeys>;
+
+/**
+ * @group Form.io components
+ * @category Base types
+ */
+export interface BaseTextareaComponentSchema extends Omit<TextareaInputSchema, 'hideLabel'> {
+  type: 'textarea';
+  // additional properties
+  showCharCount?: boolean;
+  autocomplete?: string;
+  // OF custom properties
+  rows: number;
```

Comment:
I think I'm missing:

* `autoExpand: boolean`
* `disabled`: boolean

The `rows` is not an OF custom property, but part of Formio actually.

Reply:
Do you mean have `disabled` required? It is already there but optional.

I don't find anything related to `autoExpand` in the current builder. Is it required as well or `?`llable?

Reply:
> Do you mean have disabled required? It is already there but optional.

I missed that it was part of the base type :grimacing: 

`autoExpand` is a checkbox, so you can make it required - `false` or `true` with a default of `false` in the builder: https://github.com/open-formulieren/open-forms/blob/master/src/openforms/js/components/form/textarea.js#L14